frontend/e2e/pages/protected/apply/adult-child-page.ts

```diff
@@ -1,5 +1,39 @@
+import type { Page } from '@playwright/test';
+
 import { BasePage } from '../../base-page';
 
+interface FillOutAddressArgs {
+  address: string;
+  city: string;
+  country: string;
+  page: Page;
+  postalCode: string;
+  province: string;
+}
+
+interface fillApplicantInformationFormArgs {
```

Comment:
Nitpick: to match naming conventions for interfaces

```suggestion
interface FillApplicantInformationFormArgs {
```

Same comment for other pages.